fjord/feedback/static/js/android_feedback.js

```diff
@@ -0,0 +1,66 @@
+(function($) {
+  'use strict';
+
+  // settings
+  var browserReviewUrl = 'https://play.google.com/store/apps/details?id=org.mozilla.CHANNEL#details-reviews';
+  var deviceReviewUrl = 'market://details?id=org.mozilla.CHANNEL';
+
+  // initialize
+  var reviewUrl, lastUrl = '';
```

Comment:
Style nit, it's generally preferred to put variable declarations on separate lines with separate vars. It makes it more obvious that, in this case, `reviewUrl` is `undefined` rather than an empty string.